buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -96,8 +96,8 @@ object Config {
     }
 
     object Compose {
-        const val COMPOSE_VERSION = "1.2.0-rc03"
-        const val COMPOSE_COMPILER_VERSION = "1.2.0"
+        const val COMPOSE_VERSION = "1.3.0"
```

Comment:
@LuisRocketRangel

1. Have you tested the compose UI manually?
2. Please address any deprecations mentioned in the Compose code (if any)

Reply:
Hey I re upped the PR, check it out!